WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -209,9 +209,6 @@ private extension SettingsViewController {
     private var storeSettingsRows: [Row] {
         var result = [Row]()
         if canCollectPayments {
-            result.append(.cardReadersV2)
```

Comment:
We don't want to delete this

Reply:
Yes, we do 😄 . This is part of the new "In Person Payments tab"

Reply:
Ah, so the connection flow will still be served by that. I see.

I withdraw my complaint. Carry on.